static/styles/modal.css

```diff
@@ -310,3 +310,19 @@
         font-size: 18px;
     }
 }
+
+.micromodal select {
+    height: 30px;
+    width: 220px;
+    padding: 4px 6px;
+    color: hsl(0, 0%, 33%);
+    border-radius: 4px;
+    border: 1px solid hsl(0, 0%, 80%);
+    cursor: pointer;
+    background-color: hsl(0, 0%, 100%);
```

Comment:
I think this would be better done as a `.micromodal .modal-select` CSS class -- since having styling that's controlled by the element type makes it hard to do something different for an individual component. We can do that refactor as a follow-up, though; this is clearly an improvement as written, since it scopes things in a controlled fashion.

Reply:
It's also possible the best answer for this is to just make a `settings_select` class or whatever, and have that class be usable in modals as well as settings.